packages/model/src/event.ts

```diff
@@ -8,7 +8,9 @@ export interface EventResponse {
   id: string;
   title: string;
   startDate: string;
+  startDateTimeZone: string;
```

Comment:
I thought the TZs wouldn't propagate directly to the FE

Reply:
with the new approach suggested by @peterstarling they do, this would be something like `"Europe/Lisbon"` afaik